meta-lhg/recipes-browser/chromium/chromium-ozone-wayland_%.bbappend

```diff
@@ -14,6 +14,7 @@ DEPENDS_append = " ${@bb.utils.contains('PACKAGECONFIG', 'use-ocdm', 'ocdmi', ''
 python add_ocdm_patches() {
     srcdir = d.getVar('WORKDIR', True)
     d.appendVar('SRC_URI', " file://" + srcdir + "/ocdm/patch/add_ocdm_keyssystems.patch")
+    d.appendVar('SRC_URI', " file://" + srcdir + "/ocdm/patch/add_playready_keysystem.patch")
```

Comment:
It wouldn't hurt adding this patch now, but I would prefer enable playready in open-content-decryption-module(.git) first.
ATM there is nothing there which defines OCDM_USE_PLAYREADY, and the playready stuff in opencdm_key_systems.cc is always "commented out". I can create the patch against open-content-decryption-module tomorrow morning for you to review/test (the involved files would be opencdm.gni and src/browser/BUILD.gn)

Reply:
@petegriffin BTW, could you point me at the chromium-wayland_%.bbappend snippet which set the "playready"  GYP flag in morty playready builds?
Thanks!

Reply:
> @petegriffin BTW, could you point me at the chromium-wayland_%.bbappend snippet which set the "playready" GYP flag in morty playready builds?
@andrey-konovalov Can you see this link OK? https://lhg-review.linaro.org/gitweb?p=lhg/meta-lhg-prop.git;a=blob;f=recipes-browser/chromium/chromium%25.bbappend;h=1c60cc4e49ee02d30a4d077a557eedffadd2d6e2

Reply:
> It wouldn't hurt adding this patch now, but I would prefer to enable playready in open-content-decryption-module(.git) first.
> ATM there is nothing there which defines OCDM_USE_PLAYREADY, and the playready stuff in opencdm_key_systems.cc is always "commented out". I can create the patch against open-content-decryption-module tomorrow morning for you to review/test (the involved files would be opencdm.gni and src/browser/BUILD.gn)

Ah yes I have this change locally already as part of my build, but didn't push it yet. I will do that now